wiki/Beatmap/id.md

```diff
@@ -1,12 +1,118 @@
+
+```
+https://osu.ppy.sh/beatmapsets/{BeatmapSetID}#{GameMode}/{BeatmapID}`
+```
+
+### Kategori beatmap
+
+*Artikel utama: [Kategori beatmap](Category)*
+
+Terdapat beberapa kategori pada beatmap yang telah diunggah, yang dapat berubah seiring berjalannya waktu:
+
+- [Graveyard](Category#graveyard)
+- [Work in Progress / Pending](Category#work-in-progress-and-pending)
+- [Qualified](Category#qualified)
+- [Ranked](Category#ranked)
+- [Approved](Category#approved)
+- [Loved](Category#loved)
+
+Beberapa kategori mempunyai [kriteria submisi](/wiki/Ranking_Criteria) dan memungkinkan beatmap memiliki [papan peringkat](#leaderboards) sementara atau permanen. Cara yang paling disukai untuk mendapatkannya adalah melalui [prosedur beatmap ranking](/wiki/Beatmap_ranking_procedure), sedangkan opsi kedua adalah dengan menjadikan beatmap [loved](Category#loved).
+
+## Mengunduh beatmap
+
+Beatmap dapat didapatkan melalui beberapa cara yang berbeda:
+
+### Daftar beatmap
+
+Beatmap dari berbagai kategori tersedia pada [daftar beatmap](https://osu.ppy.sh/beatmapsets) di situs web. Beatmap-beatmap tersebut dapat diurutkan berdasarkan dengan keinginan dan kebutuhan pengguna (mis. beatmap [osu!taiko](/wiki/Game_mode/osu!taiko) yang dibuat oleh mapper tertentu), lalu diunduh satu per satu. [osu!supporter](/wiki/osu!supporter) memiliki beberapa filter tambahan yang tidak dimiliki oleh pengguna biasa, seperti fitur untuk melihat beatmap di mana mereka mencapai peringkat tertentu.
+
+### osu!direct
+
+osu!direct merupakan daftar beatmap yang dibuat di dalam klien gim, yang digunakan sebagai akses cepat sekaligus menyediakan serangkaian fitur yang sama seperti daftar beatmap yang ada di situs web. Fitur ini hanya tersedia bagi pengguna [osu!supporter](/wiki/osu!supporter).
+
+### Paket beatmap
+
+*Artikel utama: [Paket beatmap (*Beatmap packs*)](Packs)*
+
+Beatmap yang ranked di bulan yang sama, atau mempunyai tema yang sama (mis. artisnya sama), dikemas untuk diunduh secara massal ke dalam arsip, disebut sebagai paket beatmap (*beatmap packs*). Daftar semua paket beatmap resmi tersedia di [situs web](https://osu.ppy.sh/beatmaps/packs).
+
+### Sumber tidak resmi
+
+Terdapat sumber-sumber beatmap lainnya, yang **tidak diperiksa atau dioperasikan oleh osu!team**. Namun, sumber-sumber tersebut tersedia di forum dan diterima oleh komunitas. Sumber-sumber tersebut meliputi:
+
+- mirror beatmap pihak ketiga yang dihosting di luar platform;
+- paket dan kompilasi beatmap yang dibuat oleh pengguna biasa yang dibagikan melalui [p2p-networks](https://en.wikipedia.org/wiki/Peer-to-peer "Wikipedia"), seperti BitTorrent.
```

Comment:
`p2p-networks` --> either `jaringan-jaringan peer-to-peer` or `jaringan-jaringan p2p`

Reply:
changed to `jaringan-jaringan peer-to-peer`